Documentation/user-manual.txt

```diff
@@ -1407,6 +1407,91 @@ size of the _CSV_ file might cause problems. Importing 100 dives at a time
+ 
+image::images/DiveSitesTab.jpg["FIGURE: Dive sites tab",align="center"]
+
+If a description for a dive site has previously been entered, this is shown in the panel
+as well as the number of dives at each site. The list of sites can be ordered by clicking
+on any of the three headings. A dustbin and an edit icon are on the left of each dive site name.
+The following actions are possible for managing the collection of dive sites:
+
+==== Filter the list of dive sites
+
+At the top right of the dive site management panel is a text box (see image above). If you wish to only show dives
+from a particular site, then type a partial dive site name such as "Genova". This shows only the dive sites
+that have been described for Genova, assuming that the word "Genova" forms part of the dive site name
+or description.
+
+==== Edit a dive site
+
+The information for each dive site can be edited either from from within the *Notes* tab or
+within the *Dive sites* tab.  In order to edit the dive site information, select the
+Edit icon to the left of the dive site name (see image above). Editing dive site information
+from within the *Notes* tab is discussed in the section on xref:S_locations[Locations] where
+each of the fields in the dive site edit panel is explained.
+
+image::images/DiveSiteEdit.jpg["FIGURE: Dive site edit",align="center"]
+
+Add any desirable information to the tab. Editing of coordinates is explained in the
+section dealing with xref:S_locations[Locations]. Having finished editing the dive site information,
+select the Done button at the top. This returns you to the dive sites management panel.
+
+==== Merge dive sites
+
+Merging two or more dives is achieved in the dive site edit panel (see image above). We assume that the dive
+sites that need merging are not far apart. Merging is performed by selecting a dive site and then merging
+other dives with this selected dive site. In the edit panel, above, select the text box labelled "Show dive sites
+in the range of:" and provide a critical distance that would include the site that needs to be merged with the present site.
+A list of sites within this range is presented (see image below).
+
+image::images/DiveSiteMerge.jpg["FIGURE: Dive sites merge",align="center"]
+
+Select the dive site to be merged by right-clicking it. A confirmation
+message is presented (see image above). Clicking the confirmation message merges the selected dive with
+the dive named at the top of the panel. Clicking the confirmation also returns you to the dive sites management panel.
+
+==== Add a dive site
+
+At the top right of the dive sites management table is a round button with a "+". Clicking that button inserts a
+new site entitled "New dive site" into the list (image below). Edit this new site by selecting its edit icon and provide a name,
+coordinates and, if preferred, a description (if required, the new site can also be deleted, e.g. after erroneously
```

Comment:
Not sure if this should be stated here - but you can also simply undo with CTRL-Z.

Reply:
When I add a new dive site and hit Ctrl-Z, the new site is not removed, so that operation is not undone. I am probably misunderstanding you. Please explain?

Reply:
Yes, the problem is that as long as a text-field is active, CTRL-Z goes only to that text field.

Try first ESC to remove focus from the text field and then undo. Alternatively, you can directly select Undo in the application-menu.

I'm not sure if this is a Qt problem or if we're doing something wrong. One day I'll have to analyze how other Qt applications behave.